aws-aurora-postgresql.yml

```diff
@@ -0,0 +1,114 @@
+version: 1
+name: csb-aws-aurora-postgresql
+id: 36203e40-2945-11ed-8980-eb81bd131a02
+description: Beta - Amazon Aurora for PostgreSQL
+display_name: Amazon Aurora for PostgreSQL (Beta)
+image_url: file://service-images/csb.png
+documentation_url: https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.2/csb-aws/GUID-main.html
```

Comment:
Sorry I think there's been a clash with my PR - the documentation_url does not work here

Reply:
I've fixed a typo in the URL, but I don't expect the URL to work until v1.2 docs go live. It's the same base URL as used for all the other services.